### DISCUSSION TITLE
Ray framework not able to read the results back

### DISCUSSION THREAD

**Discussion by nrudrappa-godaddy** [PhonePe/mantis#40] [Troubleshooting]
I am trying to run Mantis in the distributed scanning way. Scans are getting distributed in the worker and master node however the results are not getting read once the scan is completed.
I am getting the message "INFO: Awaiting results" and the scan is completed.

**Comment by Prateek-Thakare** [comment]
@nrudrappa-godaddy 
Can you please check in db if the results are updated? The worker nodes just return success failure status and not results, the results are updated directly to db from the worker nodes itself. For this to work, there must be a connectivity between the db and worker nodes too.

**Comment by nrudrappa-godaddy** [comment]
When i run as the single system it works fine. All those places where API is being called like the sslmate its working fine. Just in those cases where file needs to be created like nuclei its failing.

This is because log files are not getting created. I am assuming so.

Can you confirm if the below command works for you. test workflow has got just nuclei enabled.
python3.9 launch.py scan -o phonepe --sub phonepe.com --workflow test  -r

**Reply by Prateek-Thakare** [reply]
Do you want to scan the whole org or just a subdomain? 
The --sub flag basically finds the subdomain to scan. Maybe can you try skipping that flag if you need to scan the whole org?
